.packit.yaml

```diff
@@ -0,0 +1,40 @@
+---
+downstream_package_name: python-logdetective
+upstream_package_name: logdetective
+specfile_path: python-logdetective.spec
+upstream_tag_template: 'v{version}'
+srpm_build_deps:
+- poetry
+actions:
+  post-upstream-clone:
+    # rawhide specfile will be the source of truth: we don't need to have the specfile upstream
+    - curl -Lvso python-logdetective.spec https://src.fedoraproject.org/rpms/python-logdetective/raw/main/f/python-logdetective.spec
+  create-archive:
+    - poetry build -f sdist
+    - bash -c "ls -1t ./dist/*.tar.gz | head -n 1"
+  get-current-version:
+    - poetry version -s
+
+jobs:
+  - job: propose_downstream
+    trigger: release
+    dist_git_branches:
+      - fedora-all
```

Comment:
We do not want to push to all branches. The deps will be very likely broken in old branches.

Reply:
dropped propose_downstream in favor of pull_from_upstream